base-android/src/main/java/com/sfaxdroid/base/utils/TagUtils.kt

```diff
@@ -0,0 +1,17 @@
+package com.sfaxdroid.base.utils
+
+class TagUtils {
```

Comment:
fun getTagName() = mapOf("Nature" to "3078326", "Cars" to "4361321", ect,,,,,,,,,,,,,,,,,,,,,,,,)

Reply:
no need init and other